package-res/resources/web/dataapi/models-svc.js

```diff
@@ -55,7 +55,7 @@ pentaho.pda.SvcHandler.prototype.getSources = function(callback, options) {
             var contextName = options['context'];
             var query = "domainName=&"+((contextName)? 'context='+contextName : "");
             var jsonStr = pentahoPost( url, query, null, 'text/text' );
```

Comment:
I'd go ahead with that change here, in this call in this file. 
Not a global replacement of pentahoPost everywhere.